app/status/utils.py

```diff
@@ -1,4 +1,5 @@
 import os
+from requests.exceptions import ConnectionError
```

Comment:
It'd be better to rely on exceptions from the `dmutils.apiclient`, since requests should be an implementation detail at this point. It could be as simple as importing the `requests.exceptions.ConnectionError` in `dmutils.apiclient` and using `from apiclient import ConnectionError` here. But I guess we can change this later, since updating the utils version across repositories is a bit of a chore right now.